test/api/v3/integration/groups/POST-groups_groupId_join.js

```diff
@@ -0,0 +1,51 @@
+import {
+  generateUser,
+} from '../../../../helpers/api-integration.helper';
+
+describe('POST /group/:groupId/join', () => {
+  let user;
+
+  beforeEach(() => {
+    return generateUser().then((generatedUser) => {
+      user = generatedUser;
+    });
+  });
+
+  context('Guilds', () => {
+    it('inviting to guild doesn\'t give basilist to inviter', async () => {
+      let groupName = 'Test Guild';
+      let groupType = 'guild';
+
+      user = await generateUser({balance: 1, 'items.quests.basilist': 0 })
+      let guild = await user.post('/groups', {
+        name: groupName,
+        type: groupType
+      });
+
+      let invitedUser = await generateUser({ 'invitations.guilds': [{ id: guild._id}]});
+      await invitedUser.post('/groups/' + guild._id + '/join');
+      
+      let changedUser = await user.get('/user')
+      expect(changedUser.items.quests.basilist).to.equal(0);
+    });
+  });
+
+  context('Parties', () => {
+    it('gives basilist quest item to the inviter', async () => {
+      let groupName = "Test Party";
+      let groupType = "party";
+
+      user = await generateUser({'items.quests.basilist': 0 });

```

Comment:
I don't think this user should be instantiated with `'items.quests.basilist': 0`. The test should cover that it handles users who do not have the scroll at all yet.

In addition, there should be an additional test that has a user that is instantiated with `'items.quests.basilist': 5` (the 5 is arbitrary, it should just be greater than 0) to show that it increments the value